booking-engine-api/operations/reservations.md

```diff
@@ -77,3 +77,165 @@ Get a price quotation for a specific hotel, date interval, room category and per
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `OccupancyPrices` | array of [Room occupancy availability](hotels.md#room-occupancy-availability) | required | Pricing information. |
+
+
+
+## Get reservation price
+
+Get a final price of requested reservation
```

Comment:
I just changed it to plural "Get a total price for the requested reservations." since it calculates total price of multiple reservations withing one reservationGroup ( I know, the namespace of the endpoint is wrong :| )